{"name": "test_login_0_failed_wrong_datas[data0]", "status": "passed", "attachments": [{"name": "log", "source": "21ceb56f-03bd-4f9e-899d-d6fd7f7fc095-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data", "value": "{'user': '18684720553', 'passwd': '', 'check': '请输入密码'}"}], "start": 1608217367100, "stop": 1608217367389, "uuid": "500715cc-2718-4b16-85ad-41a6cea082a2", "historyId": "98ee763b71a2e5d2e8433e177b71f895", "testCaseId": "735933ecd585cb26d239d15346a80fa1", "fullName": "testcase.test_1login_pytest.TestLogin#test_login_0_failed_wrong_datas", "labels": [{"name": "tag", "value": "login"}, {"name": "tag", "value": "@pytest.mark.usefixtures('refresh_page')"}, {"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_1login_pytest"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-J6CF24D"}, {"name": "thread", "value": "56724-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_1login_pytest"}]}